src/builtins/BuiltinTemporal.cpp

```diff
@@ -591,6 +598,125 @@ static Value builtinTemporalDurationPrototypeAbs(ExecutionState& state, Value th
     return TemporalDuration::createTemporalDuration(state, std::abs(duration->getYear()), std::abs(duration->getMonth()), std::abs(duration->getWeek()), std::abs(duration->getDay()), std::abs(duration->getHour()), std::abs(duration->getMinute()), std::abs(duration->getSecond()), std::abs(duration->getMillisecond()), std::abs(duration->getMicrosecond()), std::abs(duration->getNanosecond()), nullptr);
 }
 
+static Value builtinTemporalPlainYearMonthConstructor(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    if (!newTarget.hasValue()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, ErrorObject::Messages::New_Target_Is_Undefined);
+    }
+
+    int referenceISODay = 1;
+
+    if (argc >= 4) {
+        referenceISODay = argv[3].asInt32();
+    }
+
+    return TemporalPlainYearMonth::createTemporalYearMonth(state, argv[0].asInt32(), argv[1].asInt32(), TemporalCalendar::toTemporalCalendarWithISODefault(state, argc >= 3 ? argv[2] : Value()), referenceISODay, newTarget);
+}
+
+static Value builtinTemporalPlainYearMonthFrom(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    auto options = Temporal::getOptionsObject(state, argc == 2 ? argv[1] : Value());
```

Comment:
Do we need to create an empty object?

Reply:
Yes, but I rewrite the function (`getOptionsObject`) and now it's not needed to create an object.